include/rawpheno.instructions.form.inc

```diff
@@ -273,7 +275,14 @@ function rawpheno_instructions_create_spreadsheet($project_id) {
 
         // Get header method and definition information.
         $t = rawpheno_function_header_properties($trait->project_cvterm_id);
-        array_push($instructions_data, array($t['name'], $t['method'], $t['definition']));
+
+        foreach($h as $m_i => $m) {
+          $star = ($m_i == 'name') ? '*' : '';
+          array_push($instructions_data, array($star . $m . ':', $t[$m_i]));

```

Comment:
We should hard-wrap the text. Maybe something like the following would work?
```php
// If the instructional text is short enough then we just add a single row with the category and the text.
if (strlen($t[$m_i]) < 300) {
  array_push($instructions_data, array($star . $m . ':', $t[$m_i]));
}
// Otherwise we want to hard-break the text, adding a row per chunk.
else {
  $wrapped_string = wordwrap($t[$m_i], 100, "\n");
  $chunks = explode("\n", $wrapped_string);
  foreach($chunks as $i => $chunk) {
    if ($i == 0) {
      array_push($instructions_data, array($star . $m . ':', $chunk));
    }
    else {
      array_push($instructions_data, array('', $chunk));
    }
  }
}
```
Essentially the goal would be to add multiple rows to the spreadsheet for a given `$m` based on the length of `$t[$m_i]`.

Reply:
works like a charm!

![screen shot 2018-06-18 at 10 50 52 am](https://user-images.githubusercontent.com/15472253/41550378-d1ea4c3c-72e5-11e8-9e17-ddf5202a23a7.png)